### PR TITLE
Remove blobber pools

### PR DESCRIPTION
**Pull request by Sriep**
## Fixes
Removes blobber pools from allocation pools.
## Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/0chain/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

## Associated PRs (Link as appropriate):
- blobber:
- gosdk: https://github.com/0chain/gosdk/pull/467
- system_test: https://github.com/0chain/system_test/pull/279
- zboxcli: https://github.com/0chain/zboxcli/pull/239
- zwalletcli:
- Other: ...
- Replaces https://github.com/0chain/0chain/pull/1311.

### CODE REVIEW

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | remove_bloober_pools  |
| gosdk  | remove_blobber_pools  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | blobber_pools  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

File: code/go/0chain.net/smartcontract/storagesc/allocation_pool_test.go [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package storagesc
 
 import (
+	"0chain.net/chaincore/tokenpool"
```
Comment (peterlimg) [inline code/go/0chain.net/smartcontract/storagesc/allocation_pool_test.go]: Seems not formatted, please run goimports to format

File: code/go/0chain.net/smartcontract/storagesc/readpool.go [hunk 2]
```diff
@@ -102,18 +96,13 @@ func (rp *readPool) moveToBlobber(sscKey, allocID, blobID string,
 		if value == moved {
 			break // all required tokens has moved to the blobber
 		}
-		var bi, ok = ap.Blobbers.getIndex(blobID)
-		if !ok {
-			continue // impossible case, but leave the check here
-		}
 		var (
-			bp   = ap.Blobbers[bi]
 			move currency.Coin
```
Comment (peterlimg) [inline code/go/0chain.net/smartcontract/storagesc/readpool.go]: We'd better not use var block for single variable

**Review by peterlimg — CHANGES_REQUESTED** [review]

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | remove_bloober_pools  |
| gosdk  | remove_blobber_pools  |
| zboxcli  | blobber_pools  |
| zwalletcli  | staging  |
| 0chain  | blobber_pools  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

**Review by peterlimg — APPROVED** [review]
lgtm, please fix the system test errors

**Comment by Sriep** [comment]
Replaced with https://github.com/0chain/0chain/pull/1373

**Pull request closed without merging**